decorators.rb

```diff
@@ -0,0 +1,24 @@
+require './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
```

Comment:
- [ ] Please ensure that you add a getter and setter like `attr_accessor :nameable`  between line 3 and line 4 in order for your attributes to be accessible, without this it will always throw an error as it happens. :slightly_smiling_face: